pandas/core/categorical.py

```diff
@@ -563,7 +566,7 @@ def _from_inferred_categories(cls, inferred_categories, inferred_codes,
             dtype = CategoricalDtype(cats, ordered=False)
             codes = inferred_codes
 
-        return cls(codes, dtype=dtype, fastpath=True)
+        return cls(codes, dtype=dtype)
```

Comment:
I am quite confused on what should be done for the fastpath parameters used in class method's to create a new `Categorial`. Should I replace them with `dtype` or let the `__init__` handle `CategorialDtype` construction?